hhs_oauth_server/request_logging.py

```diff
@@ -101,6 +108,8 @@ class RequestResponseLog(object):
         - user_id = Login user (or None) or OAuth2 API id. (BB2-342)
         - user = Login user (or None) or OAuth2 API username.
         - user_username = Login user (or None) or OAuth2 API username. (BB2-342)
+        - x-ratelimit-limit = Rate limit max.
+        - x-ratelimit-remaining = Remaining counter for rate limit.
```

Comment:
Might as well remove these 2 lines too.

Reply:
TY. Forgot about the comments. Just removed them.